### ISSUE TITLE
Enhancement request: Exclude folder and sub-folders

### ISSUE THREAD

**Issue by penalba** [farux/obsidian-auto-note-mover#45]
"Add Excluded Folder" works for the specified folder but not for nested sub-folders.  Maybe there's a way to do this with regex?

**Comment by distbit0** [comment]
Yeah this would be super useful. I assumed it applied to all sub folders but recently have realised that it does not :/

**Comment by distbit0** [comment]
@penalba I figured out how to do this.

1) Enable the "use regular expressions to check for excluded folder" option.
2) Use regex to define excluded folder paths. I.e. instead of:

Notes/Subject1/Topic1

Write:

Notes\/Subject1\/Topic1\/.*


This works for me.

**Comment by penalba** [comment]
Works for me too -- thanks!
